src/components/panel/panel.js

```diff
@@ -730,6 +708,25 @@ MdPanelService.prototype.newPanelAnimation = function() {
 };
 
 
+/**
+ * Wraps the users template in two elements, md-panel-container, which covers
```

Comment:
The `div` uses `class="md-panel-outer-wrapper"`. Should this be `md-panel-container` to be consistent with the JS docs ?